skyportal/facility_apis/ztf.py

```diff
@@ -228,5 +232,29 @@ def submit(request):
             "subprogram_name",
         ],
     }
-
     ui_json_schema = {}
+
+
+class ZTFMMAAPI(MMAAPI):
+
+    """An interface to ZTF MMA operations."""
+
+    form_json_schema = MMAAPI.form_json_schema
+
+    form_json_schema["properties"] = {
+        **form_json_schema["properties"],
+        "program_id": {
+            "type": "string",
+            "enum": ["Partnership", "Caltech"],
+            "default": "Partnership",
+        },
+        "subprogram_name": {
+            "type": "string",
+            "enum": ["GW", "GRB", "Neutrino", "SolarSystem", "Other"],
+            "default": "GRB",
+        },
+    }
+    form_json_schema["required"] = form_json_schema["required"] + [
+        "subprogram_name",
+        "program_id",
+    ]
```

Comment:
good!

Reply:
its good to slide these compliments in to keep my spirits only teetering on the edge of despair.